src/app/modules/shared/components/details-fields/details-fields.component.html

```diff
@@ -88,22 +88,11 @@
 
     <label class="col-12 col-sm-2">Time in:</label>
     <div class="col-12 col-sm-4">
-      <input
```

Comment:
Maybe we could put this under a Feature Toggle, so if something goes wrong we could change to the precious behavior without reverting changes.

Reply:
Sure, for the next tickets would be, now I have the idea to implement those feature toggles.